cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java

```diff
@@ -135,11 +98,53 @@ private CloudCredentialStatus verifyAccessKeySecretKeyIsAssumable(CloudCredentia
             LOGGER.warn(errorMessage, e);
             return new CloudCredentialStatus(cloudCredential, CredentialStatus.FAILED, e, errorMessage);
         }
-        try {
-            awsCredentialVerifier.validateAws(awsCredential);
-        } catch (AwsPermissionMissingException e) {
-            return new CloudCredentialStatus(cloudCredential, CredentialStatus.PERMISSIONS_MISSING, new Exception(e.getMessage()), e.getMessage());
+        if (cloudCredential.isVerifyPermissions()) {
+            try {
+                awsCredentialVerifier.validateAws(awsCredential);
+            } catch (AwsPermissionMissingException e) {
+                return new CloudCredentialStatus(cloudCredential, CredentialStatus.PERMISSIONS_MISSING, new Exception(e.getMessage()), e.getMessage());
+            }
         }
         return new CloudCredentialStatus(cloudCredential, CredentialStatus.VERIFIED);
     }
+
+    @Override
```

Comment:
as I see this method did not changed but on an other. Can we put back? This is just noise in the PR

Reply:
yes, idea rearranged the code and I didn't recognize, thanks